snuba/query/experiments.py

```diff
@@ -0,0 +1,31 @@
+from typing import Optional, Set
+
+from snuba import state
+from snuba.clickhouse.query_dsl.accessors import get_object_ids_in_query_ast
+from snuba.query.logical import Query as LogicalQuery
+
+
+def is_in_experiment(
```

Comment:
This functionalty has been used multiple times recently, so I thought I would pull it out to a common function.